wiki/Ranking_Criteria/osu!mania/en.md

```diff
@@ -1,78 +1,197 @@
-| 4K                                                     | Dance Dance Revolution/Stepmania (Single), EZ2ON (RubyMix), O2Jam U (4K)        |
-| 5K                                                     | pop'n music (5K), O2Jam U (5K), Pump it Up (Single)                             |
-| 6K                                                     | EZ2ON (StreetMix)                                                               |
-| 6K (L)                                                 | beatmania (P1)                                                                  |
-| 6K (R)                                                 | beatmania (P2)                                                                  |
-| 7K                                                     | O2Jam, pop'n music (7K)                                                         |
-| 8K                                                     | Dance Dance Revolution/Stepmania (Double), EZ2ON (SpaceMix)                     |
-| 8K (L)                                                 | beatmaniaIIDX (P1)                                                              |
-| 8K (R)                                                 | beatmaniaIIDX (P2)                                                              |
-| 9K                                                     | pop'n music, E5 - Dancing Super Station (Single).                               |
-| 10K                                                    | MuziBox, Pump It Up (Double)                                                    |
-| 12K                                                    | Keyboardmania (Light)                                                           |
-| 14K                                                    |                                                                                 |
-| 16K                                                    | beatmaniaIIDX (Double)                                                          |
-| 18K                                                    | E5 - Dancing Super Station (Double).                                            |
-| 20K                                                    |                                                                                 |
-
-Not included: Keyboardmania (Light+, Real) \[24K\] and (Double) \[48K\].
+The **osu!mania Ranking Criteria** is a set of rules and guidelines that apply to the creation of osu!mania-specific difficulties. In order to get an osu!mania-specific difficulty ranked, it is mandatory that the creation obeys to the listed criteria. While **all rules must be followed in any circumstance**, guidelines may be violated under exceptional circumstances. These exceptional circumstances must be warranted by an exhaustive explanation as of why the guideline has been violated and why not violating it will interfere with the overall quality of the creation.
```

Comment:
```suggestion
The **osu!mania Ranking Criteria** are a set of rules and guidelines that apply to the creation of osu!mania-specific difficulties. In order to get an osu!mania-specific difficulty ranked, it is mandatory that the creation obeys to the listed criteria. While **all rules must be followed in any circumstance**, guidelines may be violated under exceptional circumstances. These exceptional circumstances must be warranted by an exhaustive explanation as of why the guideline has been violated and why not violating it will interfere with the overall quality of the creation.
```
same issue on every RC page

Reply:
applied on all